app/views/posts/show.html.erb

```diff
@@ -7,6 +8,13 @@
         </div>
     </div>
     <p class="post-details-text"><%= @post.text %></p>
+
+
+    <!-- Comments Form -->
+    <%= render 'comments/form' %>
+
+    <!-- Likes Form -->
+    <%= render 'likes/form' %>
 </div>
```

Comment:
- [OPTIONAL] It would be nice to display the name of the commenter next to the comments. This will make the UI design uniform with the wireframe provided. 👍 😎 

![image](https://github.com/NalianAchei2020/RubyonRails-blog/assets/77106962/f0bab468-a84c-4355-b73a-514d7f7e1790)